pvlib/pvsystem.py

```diff
@@ -2491,6 +2492,14 @@ def max_power_point(photocurrent, saturation_current, resistance_series,
     nNsVth : numeric
         product of thermal voltage ``Vth`` [V], diode ideality factor ``n``,
         and number of serices cells ``Ns``
+    d2mutau : numeric
+        PVSyst thin-film recombination parameter that is the ratio of thickness
```

Comment:
I'd suggest we explain the usage of this parameter here.  Perhaps "PVsyst parameter for cadmium-telluride (CdTe) and amorphous-silicon (a:Si) modules that accounts for recombination current in the intrinsic layer. Ratio of thickness of …., in volts [V]. Default value indicates no recombination current and is appropriate for all other module types. When using the CEC module model coefficients, it is recommend to use the default value for ``d2mutau`` for all module types."

Copy/paste etc. to the other functions for consistency.